package.json

```
@@ -164,6 +164,11 @@
         "command": "swift.attachDebugger",
         "title": "Attach to Process...",
         "category": "Swift"
+      },
+      {
+        "command": "swift.captureDiagnostics",
+        "title": "Capture VSCode Swift Diagnostic Logs",
```

Drive-by nitpick: As as far as I know the recommended spelling is `VS Code`, https://code.visualstudio.com/brand

```suggestion
        "title": "Capture VS Code Swift Diagnostic Logs",
```

---

Good call, I did a separate pass and made all usages consistent in #846